server/schedulers/hot_region_config.go

```diff
@@ -79,9 +82,33 @@ func initHotRegionScheduleConfig() *hotRegionSchedulerConfig {
 	return cfg
 }
 
+func (conf *hotRegionSchedulerConfig) getRealtimeConf() *hotRegionSchedulerConfig {
+	return &hotRegionSchedulerConfig{
```

Comment:
```suggestion
	realConf := *conf
	...
	return &realconf
```

Reply:
It means copy lock, `go vet` will forbid it. And a new struct change will introduce too much diff.